refactor.md

```
@@ -298,11 +314,12 @@ try {
 ### 3.2 Circular Dependencies
 
 **Issue:** `callback.php` creates a circular dependency:
+
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)